src/core_plugins/metrics/public/kbn_vis_types/request_handler.js

```diff
@@ -9,11 +11,20 @@ const MetricsRequestHandlerProvider = function (Private, Notifier, config, timef
     name: 'metrics',
     handler: function (vis /*, appState, uiState, queryFilter*/) {
 
+      let timezone = config.get('dateFormat:tz', 'Browser');
```

Comment:
this is pretty much the same as: https://github.com/thomasneirynck/kibana/blob/262bcb411acc6269dbd0fba519bbee3f96b0e363/src/core_plugins/timelion/public/services/timezone.js#L6-L5

anyway we can reuse that one here?